webpack/assets/javascripts/react_app/components/InnerHTML/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import DelayedSkeleton from '../DelayedSkeleton';
+
+const DangerouslyInnerHTML = ({ html }) => {
+  const divRef = useRef(null);
+  const dispatch = useDispatch();
```

Comment:
I don't see anywhere in this file where `dispatch` is used.  No need to import / assign it